content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,125 @@
+By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right.
 
-To change a location's inventory frequency, first, go to the locations by clicking
-:menuselection:`Inventory --> Configuration --> Locations`.
+Click :guilabel:`Location` from the drop-down menu, then click :guilabel:`Apply`. The page now
+displays condensed drop-down menus of each location in the warehouse that has products in stock, and
+a cycle count can be performed for all products in that location.
 
-Then, click on a location to open the location settings. Next, click on :guilabel:`Edit`. In the
-:guilabel:`Inventory Frequency (Days)` field, set the number of days. For example, a location that
-needs an inventory count every 30 days would set the :guilabel:`Inventory Frequency (Days)` value
-to `30`. Once the value is entered, click :guilabel:`Save` to apply the setting to the location.
-Now, once an inventory adjustment is applied to this location, the next scheduled count date will
-be automatically set based on the number of days in the :guilabel:`Inventory Frequency (Days)`
-setting.
+.. tip::
+   In large warehouses with a large number of locations and high volume of products, it might be
+   easier to search for the specific location desired. To do this, from the
+   :menuselection:`Inventory Adjustments` page, click :guilabel:`Filters`. Then, click
+   :guilabel:`Add Custom Filter` to reveal a new menu to the right. Click this menu to reveal three
+   drop-downs.
 
-.. image:: cycle_counts/inventory-frequency.png
+   For the first field, click and select :guilabel:`Location` from the drop-down. For the second
+   field, leave the :guilabel:`contains` value as is. For the third field, type in the name of the
+   location that is being searched for. Click :guilabel:`Apply` for that location to appear on the
+   page.
+
+.. image:: cycle_counts/cycle-counts-filters.png
    :align: center
-   :alt: Edit a location to change the inventory frequency.
+   :alt: Applied filters and group by on inventory adjustments page.
+
+Change full inventory count frequency
+=====================================
+
+While cycle counts are typically performed per location, the scheduled date for full inventory
+counts of everything in-stock in the warehouse can also be manually changed to push the date up
+sooner than the date listed.
+
+By default, the scheduled date for inventory adjustments is always scheduled for the 31st of
+December of the current year. However, for some companies, it is crucial for them to have an
+accurate inventory count at all times. In such cases, the default scheduled date can be modified.
+
+To modify the default scheduled date, go to :menuselection:`Inventory app --> Configuration -->
+Settings`. Then, in the :guilabel:`Operations` section, locate the :guilabel:`Annual Inventory Day
+and Month` setting field, which includes a drop-down that is set to :guilabel:`31 December` by
```

Comment:
I know I previously suggested for this to all be guilabel, but I stand corrected since the day is user input while the month is a dropdown selection.
```suggestion
and Month` setting field, which includes a drop-down that is set to `31` :guilabel:`December` by
```